### PR TITLE
Vendor libpcap.a

### PR DESCRIPTION
**Pull request by brb**
Preparation PR for cross-compiling pwru for arm64.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,19 +9,16 @@ VERSION=$(shell git describe --tags --always)
...
-	unzip -o libpcap-1.10.4.zip
-	cd libpcap-libpcap-1.10.4/ && \
+libpcap/libpcap.a:
+	cd libpcap && \
 		./configure --disable-shared --disable-usb --disable-netmap --disable-bluetooth --disable-dbus --without-libnl && \
-		make && \
-		make install
+		make
 
 release:
```
Comment (jschwinger233) [inline Makefile:23]: `make libpcap.a` in the release block should change too

**Review by jschwinger233 — COMMENTED** [review]
:-1:

**Review by jschwinger233 — APPROVED** [review]
Never mind, we can fix the release target in the next PR for cross compiling.

**Comment by brb** [comment]
> Never mind, we can fix the release target in the next PR for cross compiling.

Yeah, I'm currently on it :+1:

**Pull request merged**